### PR TITLE
Add settings to Grbl, remove Load Cell C Flag

### PR DESCRIPTION
**Pull request by StephanCoetzee**
The use of the load cell is no longer configured with a compiler flag, but rather in the Grbl settings config file. 

Once Grbl 3.6 rolls out, we should add these settings (and the magazine gap checking settings) to `motion/config/grbl_eeprom_settings.json`. 

Adding this change to Grbl will not break Motion. However, if these settings are added to `motion/config/grbl_eeprom_settings.json` it will not work Grbl 3.5.

### CODE REVIEW

File: defaults.h [hunk 1]
```diff
@@ -110,6 +110,8 @@
   #define DEFAULT_COUNTS_PER_IDX 4000  //counts per encoder rev
   #define DEFAULT_FORCE_SENSOR_LEVEL 77 // 1.5V out of 5.0V=255 max
   #define DEFAULT_MAG_GAP_LIMIT 12.5 //in mm (units)
+  #define DEFAULT_MAG_GAP_ENABLED false
```
Comment (Jeff-Ciesielski) [inline defaults.h]: Why was this removed?

File: settings.h [hunk 2]
```diff
@@ -72,21 +72,21 @@ typedef struct {
...
   uint8_t step_invert_mask;
   uint8_t dir_invert_mask;
-  uint8_t stepper_idle_lock_time; // If max value 255, steppers do not disable.
+  uint8_t stepper_idle_lock_time;  // If max value 255, steppers do not disable.
   float junction_deviation;
   float arc_tolerance;
   uint8_t flags;  // Contains default boolean settings
   uint8_t homing_dir_mask;
-  float homing_feed_rate; //slow resolve sensor
+  float homing_feed_rate;  //slow resolve sensor
```
Comment (Jeff-Ciesielski) [inline settings.h:80]: Please don't make whitespace changes with functional changes.

Comment (Jeff-Ciesielski) [reply settings.h:80]: More of a comment for future PRs than a request for action.

**Comment by Jeff-Ciesielski** [comment]
LGTM

File: settings.h (hunk 2 above)
Comment (StephanCoetzee) [reply settings.h:80]: :+1:

**Comment by StephanCoetzee** [comment]
@mdicicco Can we merge this?
